v03_pipeline/lib/tasks/reference_data/update_variant_annotations_table_with_updated_reference_dataset_new_test.py

```diff
@@ -0,0 +1,109 @@
+from unittest.mock import patch
+
+import hail as hl
+import luigi.worker
+
+from v03_pipeline.lib.misc.io import write
+from v03_pipeline.lib.model import (
+    DatasetType,
+    ReferenceGenome,
+    SampleType,
+)
+from v03_pipeline.lib.paths import valid_reference_dataset_path
+from v03_pipeline.lib.reference_datasets.reference_dataset import (
+    BaseReferenceDataset,
+    ReferenceDataset,
+)
+from v03_pipeline.lib.tasks.files import GCSorLocalFolderTarget
+from v03_pipeline.lib.tasks.reference_data.update_variant_annotations_table_with_updated_reference_dataset import (
+    UpdateVariantAnnotationsTableWithUpdatedReferenceDataset,
+)
+from v03_pipeline.lib.test.mock_complete_task import MockCompleteTask
+from v03_pipeline.lib.test.mocked_dataroot_testcase import MockedDatarootTestCase
+
+TEST_SNV_INDEL_VCF = 'v03_pipeline/var/test/callsets/1kg_30variants.vcf'
+
+
+class UpdateVATWithUpdatedRDC(MockedDatarootTestCase):
+    def setUp(self) -> None:
+        super().setUp()
```

Comment:
yeah, I don't know if we want a new file for this... we should just be replacing the existing three `valid_reference_dataset_collection_path` with calls to new `valid_refernece_dataset_path`.